queries/fsharp/injections.scm

```diff
@@ -0,0 +1,11 @@
+([
+  (line_comment)
+  (block_comment_content)
+] @injection.content
+  (#set! injection.language "comment"))
+
+((line_comment) @injection.content
+  (#match? @injection.content "^///")
+  (#offset! @injection.content 0 3 0 0)
+  (#set! injection.language "xml")
+  (#set! injection.combined))
```

Comment:
Is that required? (It will parse all injections as one single document.)

Reply:
I think it is. A doctoring in fsharp looks like the following:
```fsharp
/// <summary>
/// text goes here
/// </summary>
let x = 42
```

So I need to combine all adjacent comment nodes to parse the embedded xml

Reply:
I think that can be handled with quantified queries @ribru17

Reply:
I think not sadly, due to the `lua-match` usage

Reply:
I tried this out and it wasn't allowed because of the `lua-match`